projects/angular-ui/src/lib/button/button.component.spec.ts

```diff
@@ -13,6 +13,7 @@ import {
 } from './tests/button.hostcomponent.spec';
 import { BaoIconComponent } from '../icon/icon.component';
 
+/* eslint-disable max-lines-per-function */
```

Comment:
I think you can adapt the rule to avoid this : 
something like (not tested) : 
```
{
    "rules": {
        "max-lines-per-function": ["error", { "code": 120 }]
    },
    "overrides": [
        {
            "files": ["*.spec.ts"],
            "rules": {
                "max-lines-per-function": ["error", { "code": 5000 }]
            }
        }
    ]
}
```